### PR TITLE
chore: update jest to version 22

### PR DESCRIPTION
**Pull request by SimenB**
Closes #39 

Thoughts?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ before_install:
 - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.3.2
 - export PATH="$HOME/.yarn/bin:$PATH"
 - yarn global add greenkeeper-lockfile@1
+install: yarn --ignore-engines
```
Comment (thymikee) [inline .travis.yml]: Why is this needed now?

Comment (SimenB) [reply .travis.yml]: https://github.com/facebook/jest/blob/9cbc26bab656fa9e9952242b2d733a86c6a61d43/packages/jest/package.json#L13

Comment (ntwb) [reply .travis.yml]: What about dropping Node.js 4.x support here, if Jest requires > Node.js 6.x?

Comment (SimenB) [reply .travis.yml]: Mainly because we don't really need to, and this plugin isn't necessarily bound to a single version of Jest

Comment (thymikee) [reply .travis.yml]: @ntwb Also, Jest supports Node 4 with custom environment targeting jsdom@9
